### PR TITLE
Add roundApi30 and roundApi33 tests to AlwaysOnKotlin CI workflow

### PR DESCRIPTION
**Pull request by alpergundogdu**

### CODE REVIEW

File: AlwaysOnKotlin/compose/build.gradle [hunk 1]
```diff
@@ -93,7 +93,12 @@ android {
         device = "Wear OS Large Round"
         apiLevel = 30
         systemImageSource = "android-wear"
-      }
+      },
```
Comment (yschimke) [inline AlwaysOnKotlin/compose/build.gradle:96]: comma not needed

Comment (yschimke) [reply AlwaysOnKotlin/compose/build.gradle:96]: These are all method invocations.

**Comment by yschimke** [comment]
Thanks for trying

```
   > System Image specified by roundApi33 does not exist.
     
     Try one of the following fixes:
     1. The image does not exist from android-wear for this architecture on the given apiLevel. However, other sources exist. Set systemImageSource to any of [google] to use.
     2. The system image does not presently exist for apiLevel 33. The latest available apiLevel is 30. Set apiLevel = 30 to use.
```

Let's leave the definition in for further testing, but just remove from the github workflow.

**Comment by yschimke** [comment]
Just noticed the 30 also timed out again

**Comment by alexvanyo** [comment]
If the `android-wear` image for API `33` is only available in the canary channel from `sdkmanager`, you can use a property `android.sdk.channel=3` in `gradle.properties` to allow fetching it.

**Comment by yschimke** [comment]
Will land and tweak separately

**Pull request merged**